core/src/com/unciv/logic/automation/unit/SpecificUnitAutomation.kt

```diff
@@ -185,7 +185,7 @@ object SpecificUnitAutomation {
 
         val foundCityAction = UnitActionsFromUniques.getFoundCityAction(unit, bestCityLocation)
         if (foundCityAction?.action == null) { // this means either currentMove == 0 or city within 3 tiles
-            if (unit.currentMovement > 0) // therefore, city within 3 tiles
+            if (unit.currentMovement > 0 && !unit.civ.isOneCityChallenger()) // therefore, city within 3 tiles
```

Comment:
A rare occurrence, but an AI can also be a oneCityChallanger and that may be a reason that the settler didn't found a city.